spec.bs

```diff
@@ -922,8 +922,7 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
 
 To <dfn>score and rank a bid</dfn> given an [=auction config=] |auctionConfig|, a [=generated bid=]
 |generatedBid|, a [=leading bid info=] |leadingBidInfo|, a [=string=] |decisionLogicScript|, a
-{{unsigned long}}-or-null |biddingDataVersion|, a [=boolean=] |isComponentAuction|, and an [=origin=]
-|topWindowOrigin|:
+{{unsigned long}}-or-null |biddingDataVersion|, an enum |auctionLevel|, which is "single-level-auction", "top-level-auction", or "component-auction", and an [=origin=] |topWindowOrigin|:

```

Comment:
We should probably define the enumeration in the IDL block for `runAdAuction()`, rather than inline here,  e.g., 

`enum auctionLevel { "a", "b", "c" };`

Reply:
Then you should be able to link to the dfn w/ something like `<a enum><code>auctionLevel</code></a>`.

Reply:
I am not sure it makes sense to have in IDL, since no web content ever sees it. It's 100% a spec thing.

Reply:
Ah, didn't catch that.